### PR TITLE
32/social impact cooperative

### PR DESCRIPTION
**Pull request by VinceXIV**
# Description

This work fixes issue #32 which is dependent on issue #31 . What has been done here is using the component developed in issue #31 and passing in values such that it looks like the image as provided in the design.

A few things were done to achieve this;
1. Create the social-impact-cooperative component using the `nx g c social-impact-cooperative` command
2. Modify the files in that component. This includes importing the **BannersModule** developed in issue #31 and passing values to it

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

Screenshot from 2023-02-20 09-37-49

# How Has This Been Tested?

I modified the files whose contents are displayed in the browser and made my work to be displayed in stead. I then modified the inputs I passed to the component to see if it does change what would be outputted to ensure that my work, indeed, uses the component from issue #31 and that it can be displayed. This push doesn't include those changes I made for testing purpose.

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by VinceXIV** [comment]
Hello @IanOdhiambo9

File: libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts [hunk 1]
```diff
@@ -4,10 +4,16 @@ import { LayoutModule } from '@elewa-group/elements/layout';
 import { SocialImpactPageComponent } from './pages/social-impact-page/social-impact-page.component';
 import { SocialImpactHeroSectionComponent } from './components/social-impact-hero-section/social-impact-hero-section.component';
 import { SocialImpactRoutingModule } from './social-impact.routing';
+import { SocialImpactCooperativeComponent } from './components/social-impact-cooperative/social-impact-cooperative.component';
+import { BannersModule } from '@elewa-group/features/components/banners'
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts:8]: fix order of imports

File: libs/pages/elewa/social-impact/src/lib/pages/social-impact-page/social-impact-page.component.html [hunk 2]
```diff
@@ -4,5 +4,4 @@
       <elewa-group-social-impact-hero-section></elewa-group-social-impact-hero-section>
```
Comment (LemmyMwaura) [inline libs/pages/elewa/social-impact/src/lib/pages/social-impact-page/social-impact-page.component.html:4]: Where is the component you created ??

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

File: libs/pages/elewa/social-impact/src/lib/pages/social-impact-page/social-impact-page.component.html (hunk 2 above)
Comment (VinceXIV) [reply libs/pages/elewa/social-impact/src/lib/pages/social-impact-page/social-impact-page.component.html:4]: This is the name the component; SocialImpactCooperativeComponent, it reuses the component created in #31, and basically passes inputs to that component

**Pull request closed without merging**